main.py

```diff
@@ -68,14 +68,15 @@ def add_Pokemon(pokemon: Pokemon):
     )
     find_pokemon = db.query(models.PokemonData).filter(models.PokemonData.id == pokemon.id).first()
```

Comment:
I don't think this is a valid check for POST route.
POST route shouldn't accept pokemon.id value as input.